test.rb:628:in `run': Error RuntimeError
